packages/ts/src/containers/xy-container/index.ts

```diff
@@ -408,4 +406,22 @@ export class XYContainer<Datum> extends ContainerCore {
     xAxis?.destroy()
     yAxis?.destroy()
   }
+
+  getAriaDescription (): string {
+    const ariaLabel = super.getAriaDescription()
+    if (this.config.ariaAutoLabel) {
+      let description = ''
+      // Get X and Y Domain
+      Object.values(ScaleDimension).forEach((dimension: ScaleDimension) => {
+        const domain = this.xyDomain[dimension]
+        description += `${dimension} domain ${domain?.[0]?.toFixed(2)} to ${domain?.[1]?.toFixed(2)} `
```

Comment:
`${dimension} domain ${domain?.[0]?.toFixed(2)} to ${domain?.[1]?.toFixed(2)} ` →
`${dimension} scale domain is from ${domain?.[0]?.toFixed(2)} to ${domain?.[1]?.toFixed(2)} `